frontend/src/app/supportAdmin/pageTitles.ts

```diff
@@ -0,0 +1,11 @@
+// js file so imports to cypress work
+export const addNewDevicePageTitle = "Add a new device";
+export const editDevicePageTitle = "Edit existing device";
+export const identityVerificationPageTitle =
+  "Verify/edit organization identity";
+export const orgAccessPageTitle = "Access organization account";
+export const addOrgAdminPageTitle = "Add organization admin";
+
+export const orgFacilityColumnTitle = "Organizations & facilities";
+export const devicesColumnTitle = "Devices";
+export const usersAndPatientsColumnTitle = "Users & patients";
```

Comment:
for "patients" we can import that as `PATIENT_TERM_PLURAL` from the `src/config/constants.ts` file